Mathlib/FieldTheory/Separable.lean

```diff
@@ -554,72 +554,136 @@ theorem Algebra.isSeparable_iff :
   ⟨fun _ x => ⟨Algebra.IsSeparable.isIntegral F x, Algebra.IsSeparable.isSeparable F x⟩,
     fun h => ⟨fun x => (h x).2⟩⟩
 
-variable {E : Type*} [Ring E] [Algebra F E] (e : K ≃ₐ[F] E)
+variable {E : Type*}
+
+section
+
+variable [Ring E] [Algebra F E] (e : K ≃ₐ[F] E)
```

Comment:
Is it possible to `open` Algebra everywhere and avoid having `Algebra.` at all places?

Reply:
I've tried but this does not work well, since based on current naming conventions, either one needs `Algebra.` or one needs `_root_.`. There are too many lemmas getting the same name after `open Algebra`.